pythia/models/butd.py

```diff
@@ -0,0 +1,294 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+
+import gc
+import torch
+from pythia.common.registry import registry
+from pythia.modules.encoders import ImageEncoder
+from pythia.modules.decoders import CaptioningDecoder
+from torch import nn
+
+from .base_model import BaseModel
```

Comment:
Is it possible to avoid relative import here: `from pythia.models.base_model import BaseModel`